src/comms/usb_hid.cpp

```diff
@@ -17,7 +17,7 @@ void CommsPacket::set_id(uint16_t id) {
 }
 
 void CommsPacket::set_info(uint16_t info) {
-	*reinterpret_cast<uint16_t*>(raw + TEENSY_PACKET_INFO_OFFSET) = info;
+	*reinterpret_cast<uint8_t*>(raw + TEENSY_PACKET_INFO_OFFSET) = info;
```

Comment:
this looks like a type mismatch to me (info is `uint16_t`. Is this intentional?

Reply:
Ah, good catch. I'll fix that.